code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -128,6 +130,8 @@
 	if(!cause || !imp_in)	return 0
 	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your macrobomb implant? This will cause you to explode and gib!", "Macrobomb Implant Confirmation", "Yes", "No") != "Yes")
 		return 0
+	imp_in.visible_message("<span class = 'userdanger'>[imp_in] starts beeping ominously!</span>") // oh shit son get the fuck out of there
+	sleep(150)

```

Comment:
Removing the macrobomb is not a fix BECAUSE microbombs are still a thing, halving the explosion size is.